background.js

```diff
@@ -32,7 +32,7 @@ chrome.tabs.onUpdated.addListener(function(tabId, change, tab) {
 
 chrome.tabs.query({ active: true, currentWindow: true }, function(tabs) {
   var currentTab = tabs[0];
-  if (currentTab !== undefined || currentTab !== null) return;
+  if (!currentTab || !currentTab.url) return;
```

Comment:
Good catch -- thanks. 

You can debug in the background page inspector by going to chrome://extensions and then clicking 'background page' link:

![image](https://user-images.githubusercontent.com/2043335/35456613-1b7a22da-02a4-11e8-8b8a-59a6d43a4a32.png)